### ISSUE TITLE
Python server crash - FileNotFoundError

### ISSUE THREAD

**Issue by laemtl** [aces/EEG2BIDS#27]
## To reproduce
Load a .edf file and event file from 2 different directories

## Log
```
- Writer: init started.
{'file_path': '/data/bids_imports/__BIDSVersion_1.4.0/sub-OTT174/ses-V1/ieeg/sub-OTT174_ses-V1_task-test_acq-seeg_ieeg.edf', 'bids_directory': '/data/bids_imports', 'read_only': False, 'events_tsv': '/data/bids_imports/__BIDSVersion_1.2.2/sub-DCC0001/ses-V01/ieeg/sub-DCC0001_ses-V01_task-test_acq-seeg_events.tsv', 'line_freq': '60', 'site_id': 'MNI', 'project_id': 'IEEG', 'sub_project_id': 'IEEG', 'visit_label': 'V0', 'subject_id': 'X X 13-May-1951 Anonymous', 'output_time': 'output-2021-04-23-18h05m16s', 'appVersion': '1.0.0'}
Exception in thread Thread-10:
Traceback (most recent call last):
  File "/usr/lib/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.7/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/data/loris-mri/bin/mri/python_virtualenvs/loris-mri-python/lib/python3.7/site-packages/socketio/server.py", line 680, in _handle_event_internal
    r = server._trigger_event(data[0], namespace, sid, *data[1:])
  File "/data/loris-mri/bin/mri/python_virtualenvs/loris-mri-python/lib/python3.7/site-packages/socketio/server.py", line 704, in _trigger_event
    return self.handlers[namespace][event](*args)
  File "/var/www/html/pycat/python/pycat.py", line 78, in edf_to_bids
    iEEG.Modifier(data, sio)  # Modifies data of BIDS format
  File "/var/www/html/pycat/python/libs/iEEG.py", line 168, in __init__
    TSV.Copy(data, sio)  # copies events.tsv to ieeg directory.
  File "/var/www/html/pycat/python/libs/TSV.py", line 136, in __init__
    with open(path_events_tsv, mode='r', newline='') as tsv_file:
FileNotFoundError: [Errno 2] No such file or directory: ''

```

**Comment by maltheism** [comment]
Hi @laemtl, I tested this bug today on macOS and I'm not getting the error like you did. Did you test with a development build or production build when you got this error? Also maybe related to the bug (but I'm doubtful) is that I see python v3.7 in your log instead of python v3.8.5. I made multiple directories for testing the bug with the events.tsv, the edf file, and even outputs directory to be selected from but no luck reproducing the bug. I'll try testing on Windows and report back what happens as well.

**Comment by maltheism** [comment]
Hi @laemtl, could you test that error on macOS and windows? I'm building the production build for Christine today and I won't have much time to test that on Ubuntu today. It would very helpful to know if you get that on macOS as well but don't sweat it if you don't have time.

**Comment by laemtl** [comment]
Fixed in #37
